Assignment03-Bootstrap/assets/scss/index.scss

```diff
@@ -0,0 +1,284 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif
+}
+
+@import '_colors';
+
+@mixin font-styling($font-size, $font-weight) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+}
+
+.body {
+  .navbar {
+    background-color: $pure_white;
+    box-shadow: 0px 2px 15px rgb(25 119 204 / 10%);
+    
+    .navbar-brand {
+      color: $dark-navy-blue;
+      @include font-styling(2rem, 700);
+    }    
+
+    .nav-link {
+      @include font-styling(.9rem, initial);
+    }
+
+    .nav-link:hover {
+      border-bottom: 1px solid $dark-blue;
+      color: $dark-blue;
+    }
+  }
+
+  #hero {
+    background: url(../images/hero-bg.jpeg) top center fixed;
+    height: 90vh;
+    margin-bottom: -200px;
+
+    .container {
+      color: $dark-navy-blue;
+
+      .hero-heading-2 {
+        @include font-styling(1.5rem, 400);
+      }
+    }
+  }
+
+  .why-us {
+    .content {
+      background-color: $dark-blue;
+      color: $pure-white;
+      padding: 2rem;
+
+      .content-heading {
+        margin-bottom: 2rem;
+      }
+
+      .content-text {
+        @extend .content-heading;
+      }
+
+      .learn-btn {
+        background-color: $light-black;
+        color: $pure-white;
+        text-decoration: none;
+      }
+    }
+    
+    .icon-box {
+      background-color: $pure-white;
+      box-shadow: 0px 2px 15px rgb(0 0 0 / 10%);
+      padding: 3rem 2rem;
```

Comment:
Please follow same rule. Either pixel or em/rem

Reply:
Changed pixels to rem.